src/installing-workflow/configuring-object-storage.md

```diff
@@ -89,6 +90,20 @@ The slugrunner reads credentials from the below locations on the filesystem.
 - Key: `/var/run/secrets/object/store/access-key-id`
 - Secret: `/var/run/secrets/object/store/access-key-secret`
 
+## [deis/controller](https://github.com/deis/controller)
+
+When the controller needs to launch a new buildpack application or scale one up, it uses a [replication controller](http://kubernetes.io/docs/user-guide/replication-controller/). Since the slugrunner needs to download the slug to run, it needs the object storage location of the slug and the object storage credentials.
+
+### Environment Variables
+
+The controller needs no environment variables for object storage configuration.
+
+### Credentials
+
+Since the object storage location information comes from the builder, the controller only needs access to the credentials information. The controller gets this information by accessing the `minio-user` secret directly from the Kubernetes API.
```

Comment:
Glad to hear it.

As a sidenote - the naming & configuration steps will get better & significantly easier in the next release. https://github.com/deis/deis/issues/4966 has details on the plan.